File: ServerSpace/src/main/java/Business/UnidadMedida.java
package Business;

import DataLayer.UnidadMedidaDB;
import EntityLayer.UnidadMedidaEntity;
import java.util.ArrayList;


public class UnidadMedida {
    
    public ArrayList<UnidadMedidaEntity> GetUnidadMedidaItems() {
        try {
            UnidadMedidaDB BD = new UnidadMedidaDB();
            return BD.GetUnidadMedidaItems();
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public ArrayList<UnidadMedidaEntity> GetUnidadMedida_Item(int Id) {
        try {
            UnidadMedidaDB BD = new UnidadMedidaDB();
            return BD.GetUnidadMedida_Item(Id);
        } catch (Exception ex) {
            System.out.println("ERROR " + ex);
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }
}

File: SharedApi/Space/Space.BusinessLayer/UnidadMedida.cs
using Space.DataLayer;
using Space.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.BusinessLayer
{
    public class UnidadMedida
    {
        public static List<UnidadMedidadEntity> GetUnidadMedidaItems()
        {
            UnidadMedidaDB DB = new UnidadMedidaDB();
            return DB.GetUnidadMedidaItems();
        }

        public static List<UnidadMedidadEntity> GetUnidadMedida_Item(Int32 UnidadMedidaId)
        {
            UnidadMedidaDB DB = new UnidadMedidaDB();
            return DB.GetUnidadMedida_Item(UnidadMedidaId);
        }
    }
}
